### DISCUSSION TITLE
thought: M&B 2 gun handling

### DISCUSSION THREAD

**Discussion by Shibva** [HbmMods/Hbm-s-Nuclear-Tech-GIT#683] [General]
So I was revisiting a bug that I remember getting with this mod and Mine and Blade Battlegear 

I found that it seems to be no longer occurring, but I then thought of an interesting idea. adding compact for M&B 2. 

I found that the guns can still work, but seem to not when its in the offhand slot. so what if special handling was given to the revolvers to allow them to fire with a left click if they're in a M&B2 offhand slot? it could allow for actual duel pistol wielding, it just needs the slight compat for it.
id think it be a nice lil compat to have with HBM's personal arsenal

id think if such would be added it could be made on abiding by what it would be in real life

=Pistols/revolvers
- can be equipped in the offhand slot and will fire if you right-click when its in the offhand
- can be dual-wielded, each will fire independently based on how you click your mouse
   - will have a reload priority, the gun in the mainhand will be prioritized for being reloaded and will then 
      reload the one in the offhand if the one in the mainhand is either full or unable to reload
-may have decreased accuracy if duel wielded

=Shotguns/Lauchers/special
- In real life, shotguns are held by both hands, so with that logic you would only be able to equip shotguns and other firearms that will require both hands to only be equipable if the mainhand slot
    -you also cant equip it in the mainhand slot if you have something in your offhand slot as well
- this just requires equipping limitations on such weapons, no additional handling would be needed; 
   should be simple enough
-Special pistols could also be put under this category as well.

**Comment by SoB1nary** [comment]
please hit yourself with a frying pan
